Source/WebCore/dom/Element.cpp

```diff
@@ -5646,4 +5643,17 @@ void Element::addVisibilityAdjustment(OptionSet<VisibilityAdjustment> adjustment
     ensureElementRareData().addVisibilityAdjustment(adjustment);
 }
 
+void Element::setWritingSuggestion(const String& writingSuggestion)
+{
+    ensureElementRareData().setWritingSuggestion(writingSuggestion);
+}
+
+String Element::writingSuggestion() const
+{
+    if (!hasRareData())
+        return emptyString();
```

Comment:
This is slightly inconsistent. If we allocate a new rare data, then the writing suggestion will be null string, not empty string. So this should return the null string `{ }` or we should change rare data to initialize `m_writingSuggestion` to the empty string.

Reply:
indeed, fixed!